onnxruntime/core/providers/qnn/builder/qnn_backend_manager.cc

```diff
@@ -1399,11 +1400,7 @@ Status QnnBackendManager::ExtractProfilingEventBasic(
   std::string message = GetEventTypeString(event_data.type);
   std::string unit = GetUnitString(event_data.unit);
 
-#ifndef _WIN32
-  tracelogging_provider_ep_enabled = false;
-#endif
```

Comment:
should we keep this in case it cause problem for non-Windows platform?
#ifndef _WIN32
  tracelogging_provider_ep_enabled = false;
#endif

Reply:
maybe not required? since the code is inside #ifdef _WIN32 anyway.

Reply:
yeah, I think we can just remove this ifdef.